code/obj/item/fitness.dm

```diff
@@ -251,3 +251,42 @@
 		src.add_fingerprint(user)
 		spam_flag = world.time + 2 SECONDS
 	return
+
+/obj/item/screamingtunapillow
+	name = "Tuna Pillow"
+	desc = "It's a pillow that has a tuna on it."
+	icon = 'icons/obj/plushies.dmi'
+	icon_state = "tunapillow"
+	var/spam_flag = 0
+
+/obj/item/screamingtunapillow/attack_self(mob/user as mob) //copy and pasted from duck code
+	if (spam_flag < world.time)
+		if (ishuman(user))
+			var/mob/living/carbon/human/H = user
+			if (H.sims)
+				H.sims.affectMotive("fun", 1)
+		spam_flag = 1
+		if (narrator_mode)
+			playsound(user, 'sound/vox/duct.ogg', 50, 1)
+		else
+			if (prob (50))
+				playsound(user, 'sound/effects/tunascream01.ogg', 50, 1)
+				user.visible_message("<B>THE TUNA PILLOW SCREAMS!</B>")
+			if (prob (50))
+				playsound(user, 'sound/effects/tunascream02.ogg', 50, 1)
+				user.visible_message("<B>THE TUNA PILLOW SCREAMS!</B>")
+		if(prob(1))
+			user.drop_item()
+			playsound(user, 'sound/effects/ohmygodtuna.ogg', 50, 1)
+			user.visible_message("<B>THE TUNA PILLOW WAILS A OMMINOUS MESSAGE, WHO MADE THIS THING!</B>")
```

Comment:
Currently I cut it down to just the tuna pillow speaks, what would you suggest for a message?

Reply:
On second though, I might delete it entirely. Seeing the prob comment above makes me thing that this would play *after* the original noise, and while not intented I do think it's better than just it being seperate.